src/components/footer.jsx

```diff
@@ -3,6 +3,6 @@ import React from 'react'
 export default (props) => (
   <footer className='footer p-footer'>
     &copy; FreeFeed 0.6.3 (February 8, 2016)<br/>
-    <a href="https://about.freefeed.net" target="_blank">About</a> | <a href="http://news.freefeed.net" target="_blank">Public News</a> | <a href="https://twitter.com/freefeednet" target="_blank">Twitter</a> | <a href="http://freefeed.reformal.ru/" target="_blank">Reformal: suggest new features</a>
+    <a href="https://beta.freefeed.net/about" target="_blank">About</a> | <a href="http://beta.freefeed.net/freefeed" target="_blank">News</a> | <a href="https://twitter.com/freefeednet" target="_blank">Twitter</a> | <a href="http://freefeed.reformal.ru/" target="_blank">Reformal: suggest new features</a>
```

Comment:
It has to be `/about` without a domain, we don't want to change later. And `<Link>`, not `<a>`.